CenterForOpenScience/osf.io
# Move website settings used outside of website [OSF-9000]

## Purpose
In anticipation of deleting the website directory entirely, clean up the code a little by moving settings used in places outside of the website directory to the place where it's used, or a central location.

## Changes

- Move settings out of website settings where appropriate
- Change imports of those settings

does not include:
- any path-related settings that rely on relative path such as `BASE_PATH` and `ROOT`


## QA Notes



Very similar to #8069, this should not change any user-facing things, but there is a lot of potential for error as this touches so much code. Would be really useful to test both of these at the same time, as they're both essentially "move a bunch of things around and make sure the new places are referenced correctly." If there is a full site regression happening at some point, this and #8069 will be best tested by that.

## Side Effects

None anticipated

## Ticket
https://openscience.atlassian.net/browse/OSF-9000
